packages/agoric-cli/test/snapshots/test-inter-cli.js.md

```diff
@@ -51,31 +51,35 @@ Generated by [AVA](https://avajs.dev).
 
     `Usage: agops inter bid by-price [options]␊
     ␊
-    Print an offer to bid collateral by price.␊
+    Place a bid on collateral by price.␊
     ␊
     Options:␊
-      --price [number]            bid price␊
-      --giveCurrency [number]     Currency to give␊
-      --wantCollateral [number]   Collateral expected for the currency␊
-      --collateralBrand [string]  Collateral brand key (default: "IbcATOM")␊
-      --offerId [number]          Offer id (default: "bid-978307200000")␊
-      -h, --help                  display help for command`
+      --from <address>             wallet address literal or name␊
+      --give <number>              IST to bid␊
+      --want <number>              max Collateral wanted (default: 1000000)␊
```

Comment:
There should be two parameters, a required `desiredBuy`, and an optional `wantMinimum`. The first goes in offerArgs and should be documented as a target. The second becomes the Zoe `want`, and should be documented as a refusal to accept partial completion of the order.

Reply:
ah. I was standing by for contract changes. But I can do it here first.

Reply:
done.